src/timedate/timedatectl.c

```diff
@@ -261,6 +261,15 @@ static int set_local_rtc(int argc, char **argv, void *userdata) {
         if (b < 0)
                 return log_error_errno(b, "Failed to parse local RTC setting '%s': %m", argv[1]);
 
+        if (b == 1) {
+                fflush(stdout);
```

Comment:
Here fflush is not needed, since we don't output anything to stdout beforehand. Rather, `print_status_info` should be switched to `log_warning` with a fflush call upfront.

Reply:
I will make these changes and test once